### ISSUE TITLE
latest version not working in nuxt as  "^1.6.1" this version does work

### ISSUE THREAD

**Issue by cds** [craigh411/vue-star-rating#65]
I had installed the latest version vue-star-rating on "nuxt": "^2.14.0", but it throws window reference error from the 
vue-star-rating.min.js file, then I had revert back to "vue-star-rating": "^1.6.1" this version the issue is solved for now.

**Comment by craigh411** [comment]
Hello,

Version 2 of `vue-star-rating` is for Vue 3 installs. Nuxt is currently using Vue 2, so you will need to continue to use vue-star-rating Version 1. I'm still maintaining both versions and all new features are being added to both at the same time, so you won't miss out on anything by using Version 1.

I hope that helps.

**Comment by craigh411** [comment]
When you say latest version, are you talking about Version 2.0? That's specifically for Vue 3 installs and Nuxt uses Vue 2.

**Comment by cristinafsanz** [comment]
In my case I had the same issue (window reference issue) when updating from 1.6.1 to 1.6.3. And in the JavaScript console I had this error:

```
Uncaught ReferenceError: Zepto is not defined
```

It works well with the 1.6.1 version, so I am restoring to this version for now.

**Comment by cds** [comment]
> When you say latest version, are you talking about vue-star-rating version 2.0? That's specifically for Vue 3 installs and Nuxt uses Vue 2.

I readdress the issue when I run ` npm install vue-star-rating` which install

>  1.6.3 version

 throw an error while I uninstall run  `npm uninstall vue-star-rating` to uninstall and again `npm install vue-star-rating@1.6.1` which install  vue-star-rating

> 1.6.1 version

 by reverting back / doing this, solves the error in nuxt specifically :-

```
> "nuxt": "^2.14.0",
> "@nuxtjs/auth": "^4.9.1",
> "@nuxtjs/axios": "^5.12.0",
> "@nuxtjs/pwa": "^3.0.0-beta.20",
```
https://we.tl/t-sNBqsEaen6  
have a look attached screenshot in link

**Comment by craigh411** [comment]
Thanks for the info. I'll update the build process so that it outputs a separate CommonJS files for use with bundlers, which, honestly, I should have done before.

**Comment by juanvillegas** [comment]
Same issue for us. It broke our SSR.

**Comment by SJJobs** [comment]
Sam issue for us.

**Comment by craigh411** [comment]
OK, I've just released `version 1.7.0` on npm where I've updated the build process which is now using the `vue-cli-service` to build  separate UMD and CommonJs files. I have tested this using the vue-cli to create a project and it all seems to work correctly, but I haven't tried it specifically in `nuxt`, so please let me know if you get any further issues.

**Comment by bruno2kd** [comment]
> OK, I've just released `version 1.7.0` on npm where I've updated the build process which is now using the `vue-cli-service` to build separate UMD and CommonJs files. I have tested this using the vue-cli to create a project and it all seems to work correctly, but I haven't tried it specifically in `nuxt`, so please let me know if you get any further issues.

I've tried vue-star-rating version 1.7.0 with nuxt version 2.15.8 and it is not working. I have to use the vue-star-rating version 1.6.1 version to work.
